### PR TITLE
Update tank application

### PR DESCRIPTION
**Pull request by EvaKaul**
- new parameter for batch ID
- new parameter for gas consumption
- new parameter for project information

### CODE REVIEW

**Comment by EvaKaul** [comment]
Hi @j0hannesfr, may you could review these changes within the tank application (docu in Readme.md and TIA project). Thanks a lot in advance!

File: tank application/tia-tank-application.zap16
Comment (j0hannesfr) [inline tank application/tia-tank-application.zap16:1]: batchID and gasConsumptionHeatingTank doesn't get reset with the reset trigger

**Review by j0hannesfr — CHANGES_REQUESTED** [review]
Please reset the 2 new parameters together with everything else in FB2_Filling Network 5 to make it consistent

**Review by EvaKaul — COMMENTED** [review]
made changes according to review

**Comment by EvaKaul** [comment]
@j0hannesfr, thanks for the review! Please check my fixes in tia-tank-application.zap16 and finish the review. Afterwards I can merge the code.

**Review by j0hannesfr — APPROVED** [review]
Thanks Eva, everything looks fine now!

**Pull request merged**